tests/test_compartments_saddle.py

```diff
@@ -182,7 +182,7 @@ def test_get_digitized():
     df = pd.DataFrame(
         [["chr1", 0, 10, pd.NA]],
         columns=["chrom", "start", "end", "value"],
-    ).astype({"value": pd.Float64Dtype()})
+    ).astype({"value": pd.Int64Dtype()})
```

Comment:
in the end - this should be Float64Dtype

Reply:
I thought int64dtype worked with newer version of pandas?

Reply:
value should be float-like, nit int-like. It was me who changed it to Int64Dtype in the middle of my branch because of old pandas. Float64Dtype is newer than Int64Dtype
sorry for confusion

Reply:
ah, got it-- yes the value should be nan-able float before digitization, categorical or nan-able int after